readme.md

```diff
@@ -190,6 +192,15 @@ merge(
 ) // => [{ a: true, b: true }, 'ah yup']
 ```
 
+### `arrayMerge: customFunction`
```

Comment:
This section feels weird to me, since the whole point of this options section is to explain `arrayMerge: yourOwnFunctionHere`